### DISCUSSION TITLE
Behavior of Transcoded Videos After Changing Transcode Settings

### DISCUSSION THREAD

**Discussion by coltenkrauter** [immich-app/immich#14752] [Q&A] [answered]
If I transcode some videos, and then later change the transcode settings and run the transcoding job again, what happens to the previously transcoded videos?  

1. Are the previous transcoded files overwritten or deleted?  
1. Or will the new transcoding create additional copies of the videos?  

I'm trying to understand how Immich handles multiple transcoding operations on the same source video.

**Comment by schuhbacca** [comment, marked as answer]
If you run the ALL job after switching, the originals will be deleted and replaced by the new. You shouldn't ever end up with multiple transcodes of a single video.

**Reply by coltenkrauter** [reply]
Perfect, thanks!
